requirements-testing.txt

```diff
@@ -1,8 +1,7 @@
 mock
 pytest-cov
 pytest-localserver
-pylama
-pylama_pylint
+pytest-pep8
```

Comment:
if the intention is just to fix the build, then pin pylama to the working versions instead of just removing it
pytest-pep8 doesn't lint a lot of other stuff